README.adoc

```diff
@@ -57,9 +62,16 @@ Fill the form as following:
 
 * _Credential name_ - this is the name of the credential that you’ll be referencing in the “Shared credential name” field of the Develocity integration page.
 * _Username_ - you can specify any username value here, as it will not be used by the plugin
-* _Password_ - this should be the value of the access key which is used to authenticate against the Develocity server.
+* _Password_ - this should be the value of the access key which is used during the build execution to get a short-lived access token from the Develocity server.
 It has a format of a key value pair (e.g. `host=value`)
 
+[#_short_lived_access_tokens]
+==== Short-lived access tokens
+Develocity access keys are long-lived, creating risks if they are leaked.To avoid this, users can use short-lived access tokens to authenticate with Develocity.Access tokens can be used wherever an access key would be used.Access tokens are only valid for the Develocity instance that created them.
```

Comment:
```suggestion
Develocity access keys are long-lived, creating risks if they are leaked. To avoid this, users can use short-lived access tokens to authenticate with Develocity. Access tokens can be used wherever an access key would be used. Access tokens are only valid for the Develocity instance that created them.
```